remaining packer content (#13285)
hashicorp/packer: website/content/docs/post-processors/compress.mdx

---
description: |

>   The `compress` post-processor compresses an artifact that contains files, such as files from
>   VMware or VirtualBox, into a single archive.

Reviewer: ```suggestion
  The `compress` post-processor compresses an artifact that contains multiple files, such as VMware of VirtualBox builds, into a single archive.
```
